buildenv

```diff
@@ -3,19 +3,38 @@
 #
 export ZOPEN_TYPE="TARBALL"
 export ZOPEN_TARBALL_URL="https://github.com/Cyan4973/xxHash/archive/refs/tags/v0.8.1.tar.gz"
-export ZOPEN_TARBALL_DEPS="curl gzip make coreutils"
+export ZOPEN_TARBALL_DEPS="curl gzip make coreutils zoslib"
 
 export ZOPEN_EXTRA_CFLAGS='-qnocsect'
 export ZOPEN_BOOTSTRAP=skip
 export ZOPEN_CONFIGURE=skip
 export ZOPEN_INSTALL_OPTS="install POSIX_ENV=Yes prefix=\${ZOPEN_INSTALL_DIR}" 
 export ZOPEN_NAME="xxHash-0.8.1"
-export ZOPEN_CHECK="skip"
 rm -f v0.8.1
 ln -s ${ZOPEN_NAME} v0.8.1
 
 zopen_check_results()
 {
-  return 3 # non-functional
+  FAILURE=0 
```

Comment:
Can you add a few lines explaining what your check is doing? I'm assuming it's validating that the checksums are right but a few words would be good here.

Reply:
Sure , Mike! I have added comments. Hope that helps. Thank you!